source/_components/ebusd.markdown

```diff
@@ -42,7 +42,7 @@ name:
   required: false
   default: ebusd
 monitored_conditions:
-  description: List of condition to monitor.
+  description: List of condition to monitor. Note that not all monitored_conditions listed here can be supported by your circuit. This integration maps limited set of keys to circuit specific ebusd values.
```

Comment:
List of conditions to monitor.

Reply:
Here I have tried to explain complications which I met during first attempt to setup this integration.
There is changing map of supported sensors per circuit:
https://github.com/sashao/home-assistant/blob/dev/homeassistant/components/ebusd/const.py

Reply:
Yes, that's good. Just change the first sentence as my suggestion.

Reply:
Fixed.